flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/service/operation/OperationManager.java

```diff
@@ -315,7 +315,7 @@ public ResultSet fetchResults(FetchOrientation orientation, int maxRows) {
 
         public ResolvedSchema getResultSchema() {
             OperationStatus current = status.get();
-            if (current != OperationStatus.FINISHED || !hasResults) {
```

Comment:
What's the usage of `hasResults`? It seems never used. Can we remove the variable?